sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionStateBuilder.scala

```diff
@@ -71,6 +71,7 @@ class HiveSessionStateBuilder(session: SparkSession, parentState: Option[Session
       new ResolveHiveSerdeTable(session) +:
         new FindDataSourceTable(session) +:
         new ResolveSQLOnFile(session) +:
+        new FallBackFileDataSourceToV1(session) +:
```

Comment:
we should put it in `BaseSessionStateBuilder`

Reply:
Yes, it is also in `BaseSessionStateBuilder`.